src/components/AvatarList/AvatarList.jsx

```diff
@@ -65,7 +70,16 @@ export const getAvatarSize = ({ size: sizeProp, stack, ...rest }) => {
 }
 
 export const AvatarList = props => {
-  const { children, max, className, center, stack, grid, ...rest } = props
+  const {
+    children,
+    max,
+    className,
+    center,
+    stack,
+    grid,
+    extraTooltipProps,
+    ...rest
+  } = props
   const avatars = React.Children.toArray(children)
```

Comment:
So this is something that Terrence and I talked a little about: it feels a bit odd that we're passing in children to this component, versus some kind of `renderComponent` or `renderChild` (it'd be `childView` or something in marionette land...) prop that tells it "hey, I want all the items in the collection I pass to be rendered as this component." I think I've seen that pattern elsewhere in HSDS (though maybe I'm just imagining it). Mostly I bring this up because we're `toArray`-ing here anyway, so wondering if it doesn't make sense to skip the middle-man? I don't mind one way or the other, just curious mostly

Reply:
This is a bit of an old component and don't remember all of the decisions that went into it,  I don't quite follow what you mean though, are you suggesting that instead of something like:

```jsx
<AvatarList>
	<Avatar />
	<Avatar />
	<Avatar />
</AvatarList>
```

We should have...

Reply:
Ahh I see, passing children would make sense _if_ you could put whatever you wanted there (which for example would allow you to wrap the avatars with a Tooltip right there), but really all you can pass are avatars. Changing it doesn't really gives many benefits and as awkward as the API currently is, it works so will leave it for now and make sure we don't do this kind of thing anymore.